doc/changelog.rst

```diff
@@ -4,6 +4,142 @@ Changelog
 =========
 
 
+.. _changelog-v2-8-0:
+
+
+v2.8.0
+------
+Highlights
+~~~~~~~~~~
+
+TODO: add highlights
```

Comment:
we should do the heyho tech-team, what you reckon it's a good highlight (one option per person) dance, man - like we did in other releases, I'll post that here

Reply:
Do we really need to have the highlight section ready to release rc1? I've just started asking around in https://github.com/ESMValGroup/ESMValCore/discussions/1954 I'd prefer to release rc1 without the highlight section and add them later, in an rc2 or if that is not needed, we can just cherry-pick from main. I've open an issue about v2.8 changelog so that this is not forgotten: #1952

Reply:
good tackling, man!